<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>밴드 채팅 보내기 매크로</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QGroupBox" name="account_group">
        <property name="title">
         <string>계정</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <layout class="QFormLayout" name="formLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="id_label">
                 <property name="text">
                  <string>이메일</string>
                 </property>
                 <property name="buddy">
                  <cstring>id_edit</cstring>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="id_edit">
                 <property name="placeholderText">
                  <string>이메일</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="pw_label">
                 <property name="text">
                  <string>비밀번호</string>
                 </property>
                 <property name="buddy">
                  <cstring>pw_edit</cstring>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="pw_edit">
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                 <property name="placeholderText">
                  <string>비밀번호</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_ip">
                 <property name="text">
                  <string>아이피</string>
                 </property>
                 <property name="buddy">
                  <cstring>ip_edit</cstring>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="ip_edit">
                 <property name="placeholderText">
                  <string>아이피 (비어 있으면 원래 아이피)</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QPushButton" name="validate_btn">
                 <property name="text">
                  <string>확인</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="add_btn">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>추가</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="account_delete_btn">
                 <property name="text">
                  <string>삭제</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="account_table">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="keyword_group">
      <property name="title">
       <string>키워드</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="keyword_edit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="content_group">
      <property name="title">
       <string>내용</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPlainTextEdit" name="content_edit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="log_group">
      <property name="title">
       <string>로그</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTextBrowser" name="log_view">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="current_repeat_label">
        <property name="text">
         <string>반복 횟수 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="current_id_label">
        <property name="text">
         <string>현재 아이디 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="current_ip_label">
        <property name="text">
         <string>현재 아이피 주소 : </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="run_btn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>실행</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stop_btn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>중지</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>run_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_run_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>642</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>611</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stop_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_stop_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>656</x>
     <y>642</y>
    </hint>
    <hint type="destinationlabel">
     <x>606</x>
     <y>611</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>content_edit</sender>
   <signal>textChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_content_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>598</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>664</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>validate_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_validation_account_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>643</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_add_account_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>643</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>-3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>account_delete_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_delete_account_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>643</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>id_edit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_id_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>-13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pw_edit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_pw_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>keyword_edit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_keyword_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>498</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>662</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ip_edit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_ip_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>527</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_run_clicked()</slot>
  <slot>on_stop_clicked()</slot>
  <slot>on_content_changed()</slot>
  <slot>on_validation_account_clicked()</slot>
  <slot>on_add_account_clicked()</slot>
  <slot>on_id_changed(QString)</slot>
  <slot>on_pw_changed(QString)</slot>
  <slot>on_delete_account_clicked()</slot>
  <slot>on_keyword_changed(QString)</slot>
  <slot>on_validation_chrome_clicked()</slot>
  <slot>on_ip_changed(QString)</slot>
 </slots>
</ui>
